pkg/app/up.go

```diff
@@ -90,6 +90,21 @@ var CommandUp = &cli.Command{
 			Usage: "Configure the shared memory size (megabyte)",
 			Value: 2048,
 		},
+		&cli.StringFlag{
+			Name:  "cpu",
```

Comment:
```sugeestion
    Name:  "cpus",
```

nit: naming "cpus" for the number of CPUs

Reply:
fixed